### PR TITLE
Created basic html and css project.

### PR DESCRIPTION
**Pull request by shubhambhoyar077**
1. Add linters locally and in GitHub Action.
2. Add html and css with h1 tag.
3. Add .gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,83 @@
...
+
+To run tests, open index.html in any web-browser.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@shubhambhoyar077](https://github.com/githubhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (mckent05) [inline README.md:83]: - Your readme file has some missing heading and their corresponding sections are also missing. Please make sure that all headings and their corresponding sections are present in your readme 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @shubhambhoyar077, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shubhambhoyar077** [comment]
Added all heading and their corresponding sections.
Questions:-
Do we have to create every project repo. as public for upcoming project?

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @shubhambhoyar077 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - All linter checks are passing ✔️ ✔️
 - Your PR is professional 👍 👍

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mckent05** [comment]
> Added all heading and their corresponding sections. Questions:- Do we have to create every project repo. as public for upcoming project?

@shubhambhoyar077 Yes, all repos have to be public for all your projects at Microverse

**Pull request merged**